src/org/loklak/harvester/Post.java

```diff
@@ -35,8 +37,16 @@ public void setTimestamp() {
         this.setTimestamp(timestamp);
     }
 
-    //public abstract void getPostId();
+    public Date getTimestampDate() {
+        return new Date(this.timestamp);
+    }
+
+    //TODO: Set up TwitterTweet before setting this as abstract
```

Comment:
They doesn't seem to be needed right now. Also some of those classes are unrelated to what I am trying to achieve. So that is why, just to get `TwitterTweet` be subclass of Post, to get results at top, I have made this arrangement. 😅 
See PR #1277